packages/augur-ui/src/modules/common/global-chat.styles.less

```diff
@@ -0,0 +1,103 @@
+@import (reference) '~assets/styles/shared';
+
+.GlobalChat {
+  background: @color-dark-grey;
+  bottom: @size-4;
+  position: fixed;
+  right: @size-22;
+  z-index: 2;
+
+  button {
+    .mono-9-bold;
+
+    align-items: center;
+    background: @color-module-background;
+    color: @color-primary-text;
+    display: flex;
+    flex-direction: row-reverse;
+    text-transform: uppercase;
+
+    > svg {
+      fill: #FFFFFF;
+      height: @size-12;
+      margin-left: @size-8;
+      transform: rotate(180deg);
+      width: @size-12;
+    }
+  }
+
+  > div {
+    border: 1px solid #0E0E0F;
+
+    // Header section
+    > div:first-of-type {
```

Comment:
can you use color constant too

Reply:
Oops, yeah I meant to go back and fix that before creating my PR.